dao/submission.js

```diff
@@ -15,194 +15,287 @@ class SubmissionDAO extends GenericDAO {
         this.submissionCollection = submissionCollection;
         this.organizationCollection = organizationCollection;
     }
+
     // prisma is unable to join study._id
```

Comment:
Thank you for getting this done for the Prisma join. Could you remove this comment?